src/js/view/components/timeslider.js

```diff
@@ -133,20 +143,21 @@ define([
             }, this);
         },
 
-        // These are new methods
-        performSeek : function () {
+        getSeekPos: function(percent) {
+            var currentPosition = this._model.get('position');
             var duration = this._model.get('duration');
             var adaptiveType = utils.adaptiveType(duration);
-            var position;
             if (adaptiveType === 'LIVE' || duration === 0) {
-                this._api.play();
```

Comment:
Remove `=== 'LIVE'`. If `duration === 0` we cannot seek. Continue to call `this._api.play();`.